src/Compilers/CSharp/Portable/Emitter/Model/SynthesizedPrivateImplementationDetailsStaticConstructor.cs

```diff
@@ -51,16 +52,23 @@ internal override void GenerateMethodBody(TypeCompilationState compilationState,
                 body.Add(payloadInitialization);
             }
 
-            // MVID = TypeOf(PrivateImplementationDetails).Module.ModuleVersionId;
+            // MVID = System.Reflection.IntrospectionExtensions.GetTypeInfo(TypeOf(PrivateImplementationDetails)).Module.ModuleVersionId;
```

Comment:
It would be good to explain what does System.Reflection.IntrospectionExtensions.GetTypeInfo do.